docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
+  will not send logs directly to Elasticsearch, instead it will use
+  `secure_forward` to send logs to mux.
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
+  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
+  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
```

Comment:
I would also argue that it is difficult to keep changing this code back and forth, both in terms of coding, and in terms of opportunity cost - the cost of keeping this in, and the rare chance that some user will stumble upon this setting and attempt to use it, vs. the time and energy it takes to yank this out.  In the meantime, we could have been fixing other bugs, adding other necessary features, etc.